### DISCUSSION TITLE
Installation from source?

### DISCUSSION THREAD

**Discussion by atsaloli** [ananthakumaran/paisa#28] [Ideas]
Hello,

Could you please provide instructions for installing Paisa from source?

I prefer to build my own binaries.

Thanks,
Aleksey

**Comment by ananthakumaran** [comment]
Check the github release https://github.com/ananthakumaran/paisa/blob/master/.github/workflows/release.yml

**Reply by atsaloli** [reply]
Thanks!  I got a 404 Not Found.  I see you removed release.yml in commit 52dfde0

**Reply by atsaloli** [reply]
I found .github/workflows/linux-cli.yml

Thanks!
